nikolai4D/frontEndFramework
# Uc 77 create dropdown atom

Added a dropdown menu

### To test
Easiest way is to look at the branch UC-77-create-drop-down from this branch https://github.com/nikolai4D/UrbanCloud-alt1/tree/UC-77-create-drop-down, so that you can view the dropdown.
Simply once this is merged type in localhost:3001/dropdown and you should be able to view it!

If something is unclear let me know :)
